### ISSUE TITLE
This is more of a question but maybe a suggestion. (read desc)

### ISSUE THREAD

**Issue by TheShadyyOneYT** [Fire-Horse27/azalea-wood#10]
I was wondering how you came up of the idea of a different door style and trapdoor to be more run-down haunted like rather than being like a standard door with slight variations to the previous existing ones. I was going to ask if you could consider (or anyone else that is reading this) to make a resourcepack or alternate door/trapdoor to look more lively. Granted, it looks really cool! Just a thought. But once again, what a lovely idea for Azalea wood set.

**Comment by Fire-Horse27** [comment]
I wanted this wood set to be a bit more unique in some way so that it wasn't just another brown wood pallet. Other common ideas for Azalea wood are for it to be green or white, but I couldn't pull those off (plus several other mods do this very thing). So, I used the more traditional brown color but decided to make things decrepit. I might provide a resource pack that offers a "restored" version in the future, but it's not in any of my current plans. Thanks for the suggestion, though!

**Comment by TheShadyyOneYT** [comment]
yeah no problem! and nice to consider it at the very least, but once again incredible mod, its simple but yet effective, and I might consider using it in my modpack. Last thing is maybe if you got into contact with someone or if you have maybe some haunted structures could be an idea! I love this concept and it can be definitely expanded on! If you would like any concept art of what I mean by "haunted" with this mod just let me know, i can make a few structure ideas.

**Comment by Fire-Horse27** [comment]
I'm a little busy with school right now, but I might expand this in the future. Thanks for the ideas and I'm glad you enjoy the mod!

**Comment by TheShadyyOneYT** [comment]
> I'm a little busy with school right now, but I might expand this in the future. Thanks for the ideas and I'm glad you enjoy the mod!

All good, take your time, my life is about to get exponentially more busy here. Good luck with school!
